data/korath/korath havens.txt

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2022 by Saugia & Terin
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+fleet "Korsmanath A'awojs"
+	government "Korath (Civilian)"
+	names "korath"
+	fighters "korath"
+	cargo 3
+	variant 21
+		"Korsmanath A'awoj"
+		"Korath Chaser" 14
+		"Korath Chaser (Expeller)" 10
+	variant 9
+		"Korsmanath A'awoj (Strike)"
+		"Korath Chaser" 17
+		"Korath Chaser (Expeller)" 7
+	variant 6
+		"Korsmanath A'awoj (Special)"
+		"Korath Chaser (Expeller)" 13
+		"Korath Chaser (Digger)" 6
+		"Korath Chaser" 5
+	variant
+		"Korsmanath A'awoj (Blaze)"
+		"Korath Chaser (Expeller)" 15
+		"Korath Chaser" 9
+
+mission "North Korsmanath A'awojs (Natural)"
+	invisible
+	landing
+	repeat
+	to offer
+		random < 50
+		not "North Korsmanath A'awojs (Natural): active"
+	to complete
+		never
+	deadline 7 1
+	destination
+		distance 1 3
```

Comment:
What's the `to complete never` for? May as well refresh the mission if we killed the Haven, instead of waiting for some more days, no?
Or is it intended that if we kill all Havens they take a while to respawn?

The destination is to make the deadline vary between 8 and 10 days, I assume.

Reply:
To illustrate my point: if you are next to exile space, go into it, find a Haven directly and kill it, then come back, it will not respawn before 6-8 days.

Reply:
They're `npc save`, not `npc kill`. If you landed on the fake destination, the mission would complete and it would despawn the Haven. It'd be very unlikely, but it's simple enough to prevent it with the `to complete - never` addendum.

Reply:
Then we should make it npc kill, not npc save?

Reply:
`npc kill` would require you to land on the destination planet to complete the mission. If we wanted there to be the dead period before it respawns, they could just be normal NPCs, but I built these to have them (have a chance) to respawn right when you land.

Reply:
I don't actually see what the issue would be to make them despawn when you land, since it could at that moment start the mission again anyway, and add further randomness anyway, but indeed there isnt a big issue here.